addons/ondevice-actions/src/components/ActionLogger/index.tsx

```diff
@@ -8,25 +8,28 @@ interface ActionLoggerProps {
   onClear: () => void;
 }
 
-export const ActionLogger: FunctionComponent<ActionLoggerProps> = ({ actions, onClear }) => (
-  <ScrollView>
-    <ScrollView horizontal>
-      <View>
-        {actions.map((action: ActionDisplay) => (
-          <View key={action.id} style={styles.row}>
-            <View>{action.count > 1 ? <Text>{action.count}</Text> : null}</View>
-            <View style={styles.grow}>
-              <Inspect name={action.data.name} value={action.data.args || action.data} />
-            </View>
-          </View>
-        ))}
-      </View>
-    </ScrollView>
+export const ActionLogger = (
```

Comment:
Just a minor thing, the FunctionComponent import was left behind and the prettier formatting isn't applied to the file.

Regardless of this I'm happy to merge this since it's a quick fix.